[HW] 1072939

weic0813 opened the pull request:
欲解決問題:
 因老師的投影片中對 ex4 之實作說明較不明確，同學們在實作時也耗費較多時間，故在 README 中增加較為詳細之敘述

 解決方法:
 於 README.md 增加:
 ●增加 git log 使用時機及如何配合使用
 ●解決完 conflict 後須使用 continue 完成先前已下達的 merge/rebase 指令

Signed-off-by: 1072939 <s1072939@mail.ncyu.edu.tw>

weic0813 commented:
已依照老師要求於 README 中加入 ex4 須完成項目之要求

jrjang commented on README.md:
```diff
@@ -8,8 +8,16 @@ bash <(curl -s https://raw.githubusercontent.com/jrjang/ncyu-2021/ex4/scripts/ex
+
+5. 解決 conflict 後須輸入 `git rebase --continue` 或 `git merge --continue` 以完成已下達之 rebase 、 merge 指令
+
+6. 完成並 commit 後, 在 ncyu-2021-ex4 資料夾下, 執行以下指令
```
以上修改可改成

2. 切到 ncyu-2021-ex4 資料夾下

3. 交換兩筆 commit: d695e045e7f8("Example 4: 3") 和 824d1cd628dc("Example 4: 2")
[提示]
- 修改前可透過 `git log` 指令確認當前 commit 資訊，已得知後續 `git rebase -i` 後續所需之 commit 編號
- 進行 rebase 及 merge 指令時，如遇到 conflict ，使用 `git status` 指令了解 conflict 原因
- 解決 conflict 後須輸入 `git rebase --continue` 或 `git merge --continue` 以完成已下達之 rebase 、 merge 指令

4. 合併 origin/ex4-3 分支, 不要用 fast forward

5. 完成並 commit 後, 在 ncyu-2021-ex4 資料夾下, 執行以下指令

jrjang commented:
Hi @weic0813 ,

commit message 可改成

在 README.md 加入 git merge/rebase 提示
* 增加 git log 使用時機及如何配合使用
* 解決完 conflict 後須使用 continue 完成先前已下達的 merge/rebase 指令

weic0813 commented:
已依照老師所建議方式進行修改，再煩請老師過目

jrjang commented on README.md:
````diff
@@ -5,11 +5,21 @@ Example 4:
 ```
 cd $HOME
 bash <(curl -s https://raw.githubusercontent.com/jrjang/ncyu-2021/ex4/scripts/ex4-pre.sh) GITHUB_ACCOUNT
+
````
這行是多餘的, 請刪除

weic0813 commented:
已將多的空行刪除

weic0813 commented:
已刪除多的空行

jrjang commented on README.md:
```diff
@@ -7,9 +7,18 @@ cd $HOME
-3. 完成並commit後, 在 ncyu-2021-ex4 資料夾下, 執行以下指令
+3. 交換兩筆 commit: `d695e04`("Example 4: 3") 和 `824d1cd`("Example 4: 2")
+   [提示]
+   
```
此行可以刪除

Commit:
[HW] 1072939

 在 README.md 增加:
 ● 增加 git log 使用時機及如何配合使用
 ● 解決完 conflict 後須使用 continue 完成先前已下達的 merge/rebase 指令

Signed-off-by: 1072939 <s1072939@mail.ncyu.edu.tw>

weic0813 commented:
已刪除多餘空行

jrjang commented:
Good job.
